Android/app/src/main/java/com/vigbag/android/SignInFragment.kt

```diff
@@ -0,0 +1,56 @@
+package com.vigbag.android
+
+import android.os.Bundle
+import android.preference.PreferenceManager
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import com.vigbag.android.databinding.FragmentSignInBinding
+
+class SignInFragment : Fragment() {
+    private lateinit var binding: FragmentSignInBinding
```

Comment:
This is same as lateinit but we use `val` for safety.
```suggestion
    private var _binding: FragmentSignInBinding? = null

    private val binding get() = _binding!!
```